packages/data-stores/src/plans/queries/use-site-plans.ts

```diff
@@ -53,6 +53,7 @@ function useSitePlans( { siteId }: Props ): UseQueryResult< SitePlansIndex > {
 							pricing: {
 								currencyCode: plan.currency_code,
 								introOffer: unpackIntroOffer( plan ),
+								discountReasonCode: plan.cost_overrides?.[ 0 ]?.override_code,
```

Comment:
I suggest we keep this closer to what is returned from the endpoint - so to not use another name/variable. Like done for `unpackIntroOffer( plan )` above, we can create a similar helper and unpack the `cost_overrides` (into basically just camelCased form). then refer to it as `sitePlan?.pricing.costOverrides?.overrideCode`.